doc/cookbook/source/examples/gaussian_processes/sparse_gaussian_process_regression.rst

```diff
@@ -0,0 +1,49 @@
+==================================
+Sparse Gaussian Process Regression
+==================================
+
+
+A Gaussian Process is a method for bayesian non-linear non-parametric regression and classification. Sparse approximations for full GP are done to reduce computational scaling. This requires selecting an additional :math:`m` latent variables which could be a  subset of training points. This subset form a pseudo data set with inputs :math:`\mathbf{X}'` and targets :math:`\mathbf{f}'` .
```

Comment:
The inducing points can either be a subset of the training data or being optimized free form